### PR TITLE
Add actions dropdown on cases list page

### PR DESCRIPTION
**Pull request by danielrus**

### CODE REVIEW

File: src/app/cases/cases-list/cases-list.component.ts [hunk 1]
```diff
@@ -64,6 +67,34 @@ export class CasesListComponent implements OnInit, OnDestroy {
...
 
+  onActionSelected(event: any): void {
+    switch (event) {
+      case 'BASIC_EXPORT':
+        break;
+      case 'DETAILED_EXPORT':
+        break;
+      case 'CASE_EXPORT':
+        break;
+      case 'SAMPLE_EXPORT':
```
Comment (adriandumavitagroup) [inline src/app/cases/cases-list/cases-list.component.ts]: maybe constants here?

File: src/app/cases/custom-case-export/custom-case-export.component.ts [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  selector: 'app-custom-case-export',
+  templateUrl: './custom-case-export.component.html',
+  styleUrls: ['./custom-case-export.component.scss'],
+})
+export class CustomCaseExportComponent {
+  appearance = CardAppearanceOptions;
+  dataMyExports = [
+    {
+      title: 'Surveillance officer export for case data',
+      link: '/soefcd.pdf',
```
Comment (adriandumavitagroup) [inline src/app/cases/custom-case-export/custom-case-export.component.ts:14]: translations needed with : this.translateService.instant(

File: src/app/shared/action-menu/action-menu.component.html [hunk 3]
```diff
@@ -0,0 +1,27 @@
+<mat-action-list *ngIf="config">
+  <ng-container *ngFor="let item of config" [ngSwitch]="item.role">
+    <ng-template [ngSwitchCase]="'LINK'">
```
Comment (adriandumavitagroup) [inline src/app/shared/action-menu/action-menu.component.html:3]: maybe constants?

File: src/app/cases/cases-list/cases-list.component.ts [hunk 4]
```diff
@@ -64,6 +67,34 @@ export class CasesListComponent implements OnInit, OnDestroy {
...
+      case 'MERGE_DUPLICATES':
+        break;
+      default:
+        break;
+    }
+  }
+
+  exportCustomCase(): void {
+    this.dialog.open(CustomCaseExportComponent, {
+      width: '776px',
```
Comment (adriandumavitagroup) [inline src/app/cases/cases-list/cases-list.component.ts]: constant please for the width

File: src/app/cases/custom-case-export/custom-case-export.component.ts (hunk 2 above)
Comment (danielrus) [reply src/app/cases/custom-case-export/custom-case-export.component.ts:14]: this is just dummy data, it will go away

**Review by adriandumavitagroup — APPROVED** [review]

**Pull request merged**